oeps/oep-0016.rst

```diff
@@ -0,0 +1,261 @@
+==========================
+OEP-16: Bootstrap Adoption
+==========================
+
++---------------+-------------------------------------------+
+| OEP           | :doc:`OEP-16 </oeps/oep-0016>`            |
++---------------+-------------------------------------------+
+| Title         | Bootstrap Adoption                        |
++---------------+-------------------------------------------+
+| Last Modified | 2017-05-17                                |
++---------------+-------------------------------------------+
+| Author        | Andy Armstrong <andya@edx.org>            |
++---------------+-------------------------------------------+
+| Arbiter       | Dennis Jen <djen@edx.org>                 |
++---------------+-------------------------------------------+
+| Status        | Draft                                     |
++---------------+-------------------------------------------+
+| Type          | Architecture                              |
++---------------+-------------------------------------------+
+| Created       | 2017-04-18                                |
++---------------+-------------------------------------------+
+
+Abstract
+========
+
+In order to accelerate development, and provide a more consistent user
+experience for our users, Open edX will adopt Bootstrap 4 to style its web
+applications. The community will be able to extend the platform using any of
+Bootstrap's collection of components, as well as use Bootstrap theming to brand
+their instances. An Open edX-supported subset of components will be provided
+that are verified to support `WCAG 2.0`_ to ensure that all new user interfaces
+conform to the `edX accessibility guidelines`_.
+
+Technology Selection
+====================
+
+The following rubric was used to assess potential technologies:
+
+* Learning curve / prior experience
+* Options for community
+* Frameworks that are opinionated vs. flexible
+* Ability to easily build accessible front-ends
+* Ease of design-framework integration
```

Comment:
Nit: "design-framework" is confusing upon initial read. Perhaps just, "Ease of integration".